charts/argo-cd/README.md

```diff
@@ -345,7 +345,6 @@ NAME: my-release
 | crds.annotations | object | `{}` | Annotations to be added to all CRDs |
 | crds.install | bool | `true` | Install and upgrade CRDs |
 | crds.keep | bool | `true` | Keep CRDs on chart uninstall |
-| createAggregateRoles | bool | `false` | Create clusterroles that extend existing clusterroles to interact with argo-cd crds |
```

Comment:
I would not move that option to follow what our other charts implement (events and rollouts).

Also Bitnami charts offers this exact option

Reply:
Bitnami have this, however placing it here would make sense if you run umbrella with other pieces such as events, rollouts. If adjusted there you can have 1 switch for all the charts.

Reply:
@mkilchhofer are you good with @pdrastil's response or still against the change? I assume latter.